RoddyCore/src/de/dkfz/roddy/execution/io/LocalExecutionService.groovy

```diff
@@ -72,6 +74,31 @@ public class LocalExecutionService extends ExecutionService {
         return outputStream;
     }
 
+    /**
```

Comment:
The reason this is here, is that the jobstate logfile has different content when jobs are executed remotely on a cluster or locally (directly) and that I wanted to keep the original behaviour. 

But, I can also put into the RuntimeService. Nevertheless, there will be two versions, one for local job execution and one for job execution on the cluster. The alternative is to break the existing way it is done.